test/http-test.js

```diff
@@ -267,8 +267,108 @@ describe('HTTP', function() {
       isvalid: true,
       address: addr.toString(node.network),
       scriptPubKey: Script.fromAddress(addr).toRaw().toString('hex'),
-      ismine: false,
-      iswatchonly: false
+      iswitness: false,
+      isscript: false
+    });
+  });
+
+  it('should not validate invalid address', async () => {
```

Comment:
Add a comment about this being the incorrect network as the reason as to why it returns invalid